base/challenge-skeleton/README.md

```diff
@@ -0,0 +1,52 @@
+# Quickstart guide to writing a challenge:
+
+* A docker image will be built from challenge/image. For the simplest challenges, replacing challenge/image/chal will be enough.
+* Edit challenge/image/Dockerfile to change the commandline or which files should get included.
+* Try the challenge locally with `make test-docker`.
+* If you have a cluster ready, deploy the challenge with `make start`.
+  * To access it, create a port forward with `make port-forward` and connect via `nc localhost PORT` using the printed port.
+* Check out `make <tab>` for more commands.
+
+## healthcheck
+
+* The healthcheck needs to run a webserver on port 45281 that responds to /healthz requests.
+ * 200 if the challenge is healthy, otherwise an error (e.g. 400).
+ * You can find an example webserer in healthcheck/image/healthz.py.
+* The base image comes preloaded with pwntools.
+* If your exploit is written in python, just drop it in healthcheck/image/doit.py.
+
+# Directory layout:
+
+## /config
+
+The config directory holds a few configuration files:
+
+* chal.conf: Configure if the challenge should be deployed to the cluster (for batch commands), if it's publicly accessible and if it has a healthcheck.
```

Comment:
I would remove the (for batch commands), and then just make `make start` fail if this is not allowed in the configuration. I didn't know this was just for batch commands and feels a bit surprising.

Reply:
it already behaves this way, but I don't think that matters in practice. The main reason this option exists is for batch commands, at least I don't see another use case.

Reply:
removed